base/system/winlogon/lang/cs-CZ.rc

```diff
@@ -1,3 +1,13 @@
+/*
+ * PROJECT:     ReactOS Winlogon
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Czech resource file
+ * TRANSLATORS: Copyright 2005 Potapnik <jirka@studioprojekt.cz>
+ *              Copyright 2005 Filip Navara <filip.navara@gmail.com>
+ *              Copyright 2011 Kamil Hornicek <kamil.hornicek@reactos.org>
+ *              Copyright 2024 Václav Zouzalík (Venca24) <vaclav.zouzalik@seznam.cz>
```

Comment:
```suggestion
 * TRANSLATORS: Copyright 2005 Jirka Ďoubek (potapnik) <jirka@studioprojekt.cz>
 *              Copyright 2024 Václav Zouzalík (Venca24) <vaclav.zouzalik@seznam.cz>
```
Notes:
- Filip Navara was not the translator, he just committed a patch by Potapnik. [[1]](https://reactos.org/forum/viewtopic.php?p=7150#p7150) [[2]](https://www.instagram.com/potapnik/)
- Kamil Hornicek only fixed UTF-8 conversion problems.

Reply:
Kamil Horníček also changed the wording of `IDS_FAILEDACTIVATEUSERSHELL` in commit 432b05a, does it count?

Reply:
Hm, alright, let's count that.